CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+    - To optional, if the rule is reserved for specific situations.
+10. Add your test files to the "tests" directory in whichever directory you selected in the earlier step.
+
+You can separate these steps into multiple pull requests. 
+If anything seems unclear, we suggest reviewing the current rules as examples. 
+You can also ask reach the maintainers for any suggestions or general guidance on how to add new rules.
+
+### Rule Metadata
+
+We suggest minimizing comments to preserve structure in each rule. 
+However, we ask that you add the description, link, and licenses. 
+These should be after the initial "---" line of the YAML file and before the rule parameters. 
+You can use the example here for reference:
+
+```yaml
+---
+# Licenses.yml: Vale rule to correct miswritten license abbreviations
+# Source: https://github.com/jargonLint/jargonLint/vale/styles/Licenses.yml
+# License: MIT
+extension: foo
+```
+
+We ask that you choose a permissive license, with a strong preference for MIT. 
+In case, your sources use a copylefted license such as the CC BY-SA or GPLv3, you can license your contribution under the same license. 
+However, we suggest searching for a functionally similar source that is permissive instead. 
+You can also discuss your sources or view our earlier discussions in the related topic, [Listing Possible Sources for information](https://github.com/jargonLint/jargonLint/discussions/31).
+
+### Optional
+
+You can link the test files and rules across directories in the repository with GitHub actions.
+In case you are not familiar with this capability, we can fulfill this step for you.
+
+## Code Review
+
+We will review your contribution at our earliest convenience and would appreciate your patience.
+
+In general, we will not edit your commits and instead only suggest edits where necessary. 
+If you disagree with our suggestions, we are open to discussing the proposed changes.
+
+Each approved pull request by contributors will remain open until the contributor signs off or several hours have passed. 
+If you want to delay the merging of the pull request for whatever reason, explain the reason in the comments of the pull request.
+
+## How to Report a Bug
+
+If you come across a bug, we ask that you answer a few questions in the issue you open:
+1. Where did you use the linter, in your system or on an external server such as GitHub actions?
+2. What was the result you expected?
+3. What is the output instead?
+4. Have you been able to replicate the issue?
+
+## Suggestions
+
+Please make any suggestions in [Discussions](https://github.com/jargonLint/jargonLint/discussions). 
+You can find discussion topics related to your suggestion and check out other topics there.
+
+## Credit
+
+By signing off on your commits, you affirm that your commit complies with the repository terms, the [Developer Certificate of Origin](/DCO) (DCO).
+This simply ensures that you have the legal right to make the contribution.
```

Comment:
How is this? I also added the DCO to the root directory

Reply:
Great 👍